Source/Particles/ParticleCreation/FilterCreateTransformFromFAB.H

```diff
@@ -89,25 +89,47 @@ Index filterCreateTransformFromFAB (DstTile& dst1, DstTile& dst2, const amrex::B
     // For loop over all cells in the box. If mask is true in the given cell,
     // we create the particles in the cell and apply a transform function to the
     // created particles.
+#if (AMREX_SPACEDIM == 3)
```

Comment:
This looks a bit too verbose on first sight.
How about this:
```
constexpr int spacedim = AMREX_SPACEDIM;
```
then below...

Reply:
It's a very good solution indeed! I've just implemented it...

Reply:
Hurray, glad you like it :)